### PR TITLE
Added Client Server Implementation

### PR DESCRIPTION
**Pull request by Nairit11**

### CODE REVIEW

File: Simple Client Server/client.c [hunk 1]
```diff
@@ -0,0 +1,65 @@
+
+// Write CPP code here 
+#include <netdb.h> 
+#include <stdio.h> 
+#include <stdlib.h> 
+#include <string.h> 
+#include <sys/socket.h> 
+#define MAX 80
```
Comment (christianbender) [inline Simple Client Server/client.c]: Please add the header ```stdio.h``` and ```stdlib.h``` for using of various functions that you call.

**Comment by christianbender** [comment]
@Nairit11 I get by compiling ```server.c``` this compiler-errors (using Linux)
screenshot

I added the headers stdio.h (for printf) and stdlib.h

File: Simple Client Server/client.c [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
+// Write CPP code here
```
Comment (christianbender) [inline Simple Client Server/client.c]: Add header ```unistd.h``` for using of ```write``` and ```read```

File: Simple Client Server/server.c [hunk 3]
```diff
@@ -0,0 +1,94 @@
+
```
Comment (christianbender) [inline Simple Client Server/server.c]: Same as above

**Comment by Nairit11** [comment]
Though the codes run perfectly on my system, there must be some platform independence issues. I have added the suggested header files. Kindly review. Thank You.

**Comment by Nairit11** [comment]
Though the codes run perfectly on my system, there must be some platform independence issues. I have added the suggested header files. Kindly review. Thank You.

**Comment by christianbender** [comment]
@Nairit11 Thanks for the fast reaction.

**Pull request merged**